### PR TITLE
Fum 2668 - Document usage

### PR DESCRIPTION
**Pull request by rbapst-tamedia**
Document how to use cron-helm-chart
Add "suspend" feature

After merge, create a "v1.1.0" release (because of the "suspend" new feature)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,57 @@
...
+  create_namespace = true
+  max_history      = 10
+
+  values = [
+    templatefile("${path.module}/example.yaml", {
+      service_name     = "search"
+      image_tag        = "8.2.1"
+      datadog_env      = "disco-sbx"
+      url_env          = "https://www.google.com"
+      cronjob_schedule = "0/10 * * * *" # Every 10 minutes
```
Comment (sgametrio) [inline README.md]: @rbapst-tamedia [minor] shouldn't it be `*/10`?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,57 @@
...
+    }),
+    yamlencode({
+      args = ["https://www.google.com"]
+    })
```
Comment (sgametrio) [inline README.md:54]: @rbapst-tamedia What is this for? Seems a bit complex as an example 😆

File: README.md [hunk 3]
```diff
@@ -1,15 +1,57 @@
 # cron-helm-chart
-Contains the generic helm chart that run kubernetes cron job. It is used by Prometheus & Disco project
+
+Contains the generic helm chart that run kubernetes cron job. It is used by Prometheus & Disco projects
 
 ## Usage
 
-Reference the release of the chart you want to deploy in terraform
+To deploy a k8s cronjob simply create a helm value file and refer it in the terraform code.
+Here and example with a value file accepting some parameters set from the terraform code
```
Comment (sgametrio) [inline README.md]:
```suggestion
Here it is an example with a value file accepting some parameters set from the terraform code
```

File: README.md [hunk 4]
```diff
@@ -1,15 +1,57 @@
 # cron-helm-chart
-Contains the generic helm chart that run kubernetes cron job. It is used by Prometheus & Disco project
+
+Contains the generic helm chart that run kubernetes cron job. It is used by Prometheus & Disco projects
 
 ## Usage
 
-Reference the release of the chart you want to deploy in terraform
+To deploy a k8s cronjob simply create a helm value file and refer it in the terraform code.
```
Comment (sgametrio) [inline README.md:7]: @rbapst-tamedia It is true, but one can use the helm chart directly without Terraform. It's also true that it's not mandatory to create a `values.yaml` but you can pass Values directly via other means (`set {}` blocks, `values = [args=["foo", "bar"]]`, etc..)

what if, in the Readme we put the simplest example possible and then we create an `examples` folder with more complex like the one you showed us?

**Review by sgametrio — APPROVED** [review]
I added some comments @rbapst-tamedia 

If this is blocking you, you can merge anyway.

File: README.md (hunk 1 above)
Comment (rbapst-tamedia) [reply README.md]: Right. On linux 5/10 means a 05, 15, 25, ... but it looks that's implementation specific. I'll correct

**Pull request merged**